scripts/task-central.sh

```
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Central Task Management Script
+# Use this script from ANY worktree to manage tasks centrally
+
+MAIN_WORKTREE="/Users/kuoloonchong/Desktop/akarii-test"
+
+# Function to run task-master commands in main worktree
+run_task_command() {
+    cd "$MAIN_WORKTREE"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)